src/main/java/edu/fiuba/algo3/modelo/IPregunta.java

```diff
@@ -0,0 +1,12 @@
+package edu.fiuba.algo3.modelo;
+
+import java.util.List;
+
+public interface IPregunta {
```

Comment:
Si bien no está mal la convención de agregar el prefijo `I` cuando es una interface, es la convención de .net y muchos otros lenguajes, pero no es la convención en Java o no es muy usado. 
Algunas discusiones al respecto:
- https://stackoverflow.com/questions/541912/interface-naming-in-java 
- https://medium.com/@thomaspoignant/java-how-to-name-interface-and-implementor-94c0fa564b87

Reply:
Ustedes deciden si lo dejan así o lo cambian, no es muy importante es solo una observación.